CookSmart/CookSmart/IngredientListViewController.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  IngredientListViewController.swift
+//  cake
+//
+//  Created by Alex King on 3/28/20.
+//  Copyright © 2020 Olga Galchenko. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+class IngredientListViewController: UIHostingController<IngredientListView> {
+  init() {
+    super.init(rootView: IngredientListView())
+  }
+
+  @objc
+  dynamic required init?(coder _: NSCoder) {
+    assertionFailure("init(coder:) has not been implemented")
+    return nil
+  }
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    CSIngredients.sharedInstance()?.refreshRecents()
```

Comment:
Also, shouldn't this go into viewWillAppear or something like that, i.e. wouldn't we want to refresh recents every time this view is shown.